src/ci/docker/host-x86_64/x86_64-uefi/uefi_qemu_test/run.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+from pathlib import Path
+
+
+def run(*cmd, env=None, capture=False):
+    """Print and run a command, optionally capturing the output."""
+    cmd = [str(p) for p in cmd]
+    print(' '.join(cmd))
+    return subprocess.run(cmd,
+                          capture_output=capture,
+                          check=True,
+                          env=env,
+                          text=True)
+
+
+def build_and_run(tmp_dir):
+    host_artifacts = Path('/checkout/obj/build/x86_64-unknown-linux-gnu')
+    stage0 = host_artifacts / 'stage0/bin'
+    stage2 = host_artifacts / 'stage2/bin'
+
+    env = dict(os.environ)
+    env['PATH'] = '{}:{}:{}'.format(stage2, stage0, env['PATH'])
+    env['RUSTFLAGS'] = '-C linker=lld-link'
```

Comment:
The targets use `rust-lld` by default, which as far as I can tell we can't use here unless we set `NO_DOWNLOAD_CI_LLVM=1`. I avoided doing that because it would increase the build time. I think the issue would maybe be fixed by https://github.com/rust-lang/rust/pull/100010 though.

Added a comment to explain this, happy to enable `NO_DOWNLOAD_CI_LLVM` if preferred though.

Reply:
With the latest push it's now running under a docker file that does have `NO_DOWNLOAD_CI_LLVM` enabled, so this work around is no longer needed and I've removed it.